pkg/simple/client/devops/jenkins/options.go

```diff
@@ -25,10 +25,14 @@ import (
 )
 
 type Options struct {
-	Host           string `json:",omitempty" yaml:"host" description:"Jenkins service host address"`
-	Username       string `json:",omitempty" yaml:"username" description:"Jenkins admin username"`
-	Password       string `json:",omitempty" yaml:"password" description:"Jenkins admin password"`
-	MaxConnections int    `json:"maxConnections,omitempty" yaml:"maxConnections" description:"Maximum connections allowed to connect to Jenkins"`
+	Host                       string `json:",omitempty" yaml:"host" description:"Jenkins service host address"`
+	Username                   string `json:",omitempty" yaml:"username" description:"Jenkins admin username"`
+	Password                   string `json:",omitempty" yaml:"password" description:"Jenkins admin password"`
+	MaxConnections             int    `json:"maxConnections,omitempty" yaml:"maxConnections" description:"Maximum connections allowed to connect to Jenkins"`
+	DevOpsServiceAddress       string `json:"devopsServiceAddress" yaml:"devopsServiceAddress" description:"The separate DevOps Service address"`
+	DevOpsPluginServiceAddress string `json:"devopsPluginServiceAddress" yaml:"devopsPluginServiceAddress" description:"The KubeSphere DevOps plugin Service address"`
```

Comment:
What's the difference between `DevopsServiceAddress` and `DevOpsPluginServiceAddress`?

Reply:
`DevopsServiceAddress` is the address of ks-devops-apiserver from https://github.com/kubesphere-sigs/ks-devops/

`DevOpsPluginServiceAddress` is the address of ks-devops-plugin from https://github.com/kubesphere-sigs/ks-devops-plugin

Of course, this is just a temporary solution. We will remove them once the real plugin mechanism of KubeSphere is ready.